src/ecrc-frontend/src/components/page/orgVerification/OrgVerification.js

```diff
@@ -143,7 +143,6 @@ export default function OrgVerification({ page: { header, org, setError } }) {
       <div className="page">
         <div className="content col-md-8">
           <h1>Organization Information</h1>
-          <br />
```

Comment:
was this space removed from all pages between the main header and other text?

Reply:
Yes, the header and paragraph already have spacing. I did update all the pages